docs/snap-actions-playbook.md

```diff
@@ -0,0 +1,64 @@
+# Snap actions playbook
+
+This document describes how to interact with the Tailscale snap itself for performing various tasks.
+
+## How to view the service logs
+
+The logs can be viewed through `snap logs`:
+
+```bash
+sudo snap logs tailscale
+```
+
+For more lines:
+
+```bash
+sudo snap logs -n 100 tailscale
+```
+
+## How to manage the Tailscale service
+
+To stop the Tailscale service:
+
+```console
+$ sudo snap stop tailscale
+Stopped.
+```
+
+While the service is stopped, the `tailscale` client command is expected to fail with the following message:
+
+```console
+$ tailscale status
+failed to connect to local tailscaled; it doesn't appear to be running (sudo systemctl start tailscaled ?)
+```
+
+To start the Tailscale service again:
+
+```console
+$ sudo snap start tailscale
+Started.
+```
+
+The `tailscale` client command should be working again.
+
+The service can also be restarted in a single command:
+
+```console
+$ sudo snap restart tailscale
+Restarted.
+```
+
+Stopping and starting or restarting the Tailscale service should *not* result in needing to re-login.
+
+## How to update Tailscale
+
+To update Tailscale on a machine, refresh through snap:
+
+```console
+$ sudo snap refresh tailscale
+tailscale ... refreshed
+```
+
+This process will auto-restart the Tailscale service.
```

Comment:
I'm not sure if this is what we actually want though. Earlier talks with @marcusboden suggested that he would not want tailscaled to restart during a snap refresh, because this could cause unexpected network interruptions. I've opened a PR for review to disable the auto-restart on refresh: https://github.com/canonical/tailscale-snap/pull/44